### PR TITLE
First commit by provostma21

### PR DESCRIPTION
**Pull request by provostma21**
Added text to README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
-# git_repo_setup
\ No newline at end of file
+# git_repo_setup
+
+
+
+
```
Comment (analisegiobbi3) [inline README.md:5]: hey lose the crazy number of spaces

Comment (provostma21) [reply README.md:5]: no

File: README.md [hunk 2]
```diff
@@ -1 +1,21 @@
...
+
+
+
+
+
+I enjoy programming with a real coding language.
```
Comment (analisegiobbi3) [inline README.md:21]: nice!

**Review by analisegiobbi3 — APPROVED** [review]
ship it!

**Pull request merged**